[DOCS] Kibana 9.5.2  release notes (#286098)
elastic/kibana: docs/release-notes/index.md

Reviewer: ```suggestion
```
This fix was reverted before 9.5.1 shipped (
#282883, backport 
#283157). It re-lands as 
#283177, which we already document in 9.5.2 at line 61, so nothing is lost by removing it here.

Author: Addressed in https://github.com/elastic/kibana/pull/286098/commits/521ceb605e03873e662fc0e6107dca4c8c65d367.

% FEATURES, ENHANCEMENTS, FIXES
% Paste in index.md

## 9.5.2 [kibana-9.5.2-release-notes]

### Features and enhancements [kibana-9.5.2-features-enhancements]

> **Alerting and cases**:

Reviewer: ```suggestion
**Elastic Agent Builder**:
* Remove the one-time **Introducing AI Agent** modal that appeared on first load in {{elastic-sec}} and {{product.observability}}. To switch between **AI Agent** and **AI Assistant**, use the chat experience setting in **GenAI Settings** [#277048]({{kib-pull}}277048).

**Alerting and cases**:
```
The modal is gone in 9.5.2

* Add a public API to create, update, and delete case templates (`POST`, `PUT`, and `DELETE /api/cases/templates`), including a dry-run mode that validates a template without saving it [#280144]({{kib-pull}}280144).

> * Improve the case view, cases list, and field library by keeping the case details panel in view as you scroll, saving status and other fields immediately when changed, and adding drag-and-drop field reordering to the field library [#282197]({{kib-pull}}282197).

Reviewer 2: ```suggestion
* Improve the Cases view, cases list, and field library by keeping the case details panel in view as you scroll, saving status and other fields immediately when changed, and adding drag-and-drop field reordering to the field library [#282197]({{kib-pull}}282197).
```

* Enable running a workflow with only the `execute` privilege, without also requiring the `read` privilege on the workflow definition [#284822]({{kib-pull}}284822).

### Fixes [kibana-9.5.2-fixes]

**Elastic Agent Builder**:
* Fix missing screen reader announcements when toggling the **Show active only** filter on the agent tools page [#282276]({{kib-pull}}282276).
* Fix Agent Builder queries failing with a security exception on {{es}} 9.4 when you configure {{ccs}} index patterns but the cluster doesn't have the remote cluster client role [#281609]({{kib-pull}}281609).
* Fix chat completion streams continuing after you cancel a request, and cap their duration at 10 minutes [#285119]({{kib-pull}}285119).
* Fix Agent Builder requests to Google Vertex AI {{infer}} endpoints failing when tool schemas include JSON Schema fields that Vertex doesn't accept [#284810]({{kib-pull}}284810).
* Fix Agent Builder conversations failing when a prompt contains special tokens such as `<|endoftext|>`, or when Claude models receive empty text blocks [#283177]({{kib-pull}}283177).

> * Restore legacy root-level inputs in Agent Builder workflow tool schemas so existing workflow tools keep working [#285149]({{kib-pull}}285149).

Reviewer: ```suggestion
* Restore legacy root-level inputs in Agent Builder workflow tool schemas so existing workflow tools keep working [#285149]({{kib-pull}}285149).
* Fix concurrent conversations silently dropping a round when two clients replied at the same time. Case attachment requests that now conflict return HTTP 409 instead of appearing to succeed [#281887]({{kib-pull}}281887).
* Fix agent custom instructions failing to save because the full edit payload was sent, which broke access control entries [#277176]({{kib-pull}}277176).
* Fix visualizations generated from a specified index silently returning an empty chart when the query read from a different source. Agent Builder now retries once and then reports an error [#282212]({{kib-pull}}282212).
* Fix the custom answer text field losing focus after you select its radio button when an agent asks a clarifying question [#283326]({{kib-pull}}283326).
```
Adding 4 missing items.

% !!DEFERRED!! Not yet confirmed in build candidate 9.5.2-6859b2a4 (backport merged after BC cutoff). Re-verify against the next BC before publishing.
% * Keep answers to clarifying questions in the research agent's context when a later question is asked in the same round [#284800]({{kib-pull}}284800).

**Alerting and cases**:
* Fix Cases free-text search failing when a case's extended fields produced too many runtime-field values [#284475]({{kib-pull}}284475).
* Fix case Activity recording a row for every unchanged template or global field default when you create a case [#284310]({{kib-pull}}284310).
* Fix assigning users to a case creating duplicate activity entries [#284035]({{kib-pull}}284035).

> * Fix Field Library toggle filters missing from **More filters** on the **All cases** page [#283645]({{kib-pull}}283645).

Reviewer 2: ```suggestion
* Fix field library toggle filters missing from **More filters** on the **All cases** page [#283645]({{kib-pull}}283645).
```
To be consistent with other instances on the page

% * Fix cases created from v2 templates by a system action not inheriting the template's external connector, sync and observables settings, and assignees [#284854]({{kib-pull}}284854).
% * Fix case attachment UI issues, including delete controls on saved-object attachments and the more-actions menu for read-only access [#285456]({{kib-pull}}285456).
% * Fix legacy custom fields and their v2 field-definition counterparts falling out of sync, by linking each legacy field to its definition and keeping both representations equal on every write [#282772]({{kib-pull}}282772).

**Dashboards and Visualizations**:
* Fix Lens metric trendline breakdown ordering to rank by the trendline's own metric instead of falling back to alphabetical order [#283159]({{kib-pull}}283159).
* Fix histogram granularity to preserve the saved bar count instead of resetting to **Auto** above 7 bars. The effective ceiling remains the `histogram:maxBars` advanced setting [#283058]({{kib-pull}}283058).
* Fix duration format to preserve decimal and compact settings on round-trip, except when using the automatic (`auto-approximate`) duration format [#282815]({{kib-pull}}282815).

> * Fix the dashboards list endpoint not sorting by last updated date, descending, when no search query is set [#282363]({{kib-pull}}282363).

Reviewer 2: ```suggestion
* Fix the dashboards list endpoint not sorting by last updated date in descending order when no search query is set [#282363]({{kib-pull}}282363).
```

% !!DEFERRED!! Not yet confirmed in build candidate 9.5.2-6859b2a4 (not yet on the 9.5 branch). Re-verify against the next BC before publishing.
% * Fix by-reference Lens panels disappearing from dashboards when the panel's `savedObjectRef` reference is missing [#285476]({{kib-pull}}285476).

**Data ingestion and {{fleet}}**:
* Fix deleting a package policy from an agent policy with many agents blocking on a full synchronous redeploy instead of deploying asynchronously [#282580]({{kib-pull}}282580).
* Fix Fleet bulk upgrades installing the bundled package version instead of the latest registry version [#282829]({{kib-pull}}282829).
* Fix Fleet lookups that treated version-specific policy IDs as base policy IDs, which could return HTTP 404 for Fleet Server policy revisions and miss agents [#283071]({{kib-pull}}283071).

> * Fix the agentless enrollment flyout failing to detect enrolled agents assigned to version-specific policies [#283434]({{kib-pull}}283434).

Reviewer 3: ```suggestion
* Fix the managed integration enrollment flyout failing to detect enrolled agents assigned to version-specific policies [#283434]({{kib-pull}}283434).
```
